css/templates/lune/components/_chapter.css

```diff
@@ -11,9 +11,9 @@
 .lune-chapter-name,
 .lune-chapter-number {
   /* Width is one letter wide */
-  width: .7em;
+  width: 1em;
   word-wrap: break-word;
-  letter-spacing: 0;
+  letter-spacing: .12em;
```

Comment:
Added custom letter spacing here to make sure letters wrapped correctly and stays centered when on one line